service_container/autowiring.rst

```diff
@@ -547,13 +547,38 @@ under the arguments key.
 
 Another possibility is to use the ``#[Target]`` attribute. By using this attribute
 on the argument you want to autowire, you can define exactly which service to inject
-by using its alias. Thanks to this, you're able to have multiple services implementing
-the same interface and keep the argument name decorrelated of any implementation name
-(like shown in the example above).
+by passing the name of the argument used in the named alias. Thanks to this, you're able
+to have multiple services implementing the same interface and keep the argument name
+decorrelated of any implementation name (like shown in the example above).
```

Comment:
In addition, you'll get an exception in case you make any typo in the target name.

Reply:
Done